packages/dd-trace/src/appsec/iast/analyzers/path-traversal-analyzer.js

```diff
@@ -6,7 +6,19 @@ const InjectionAnalyzer = require('./injection-analyzer')
 class PathTraversalAnalyzer extends InjectionAnalyzer {
   constructor () {
     super('PATH_TRAVERSAL')
-    this.addSub('datadog:fs:access', obj => this.analyze(obj.arguments))
+    this.addSub('apm:fs:operation:start', obj => {
+      const pathArguments = []
+      if (obj.path) {
+        pathArguments.push(obj.path)
+      }
+      if (obj.src) {
+        pathArguments.push(obj.path)
```

Comment:
Shouldn't this use the same property as the condition?

Reply:
Yes! I've detected it also adding more test to path traversal.